packages/paste-website/src/pages/components/date-picker/index.mdx

```diff
@@ -0,0 +1,617 @@
+    </>
+  );
+};
+```
+
+#### DatePicker Props
+
+All the valid HTML attributes for `input` (except `type`) are supported, including the following props:
+
+| Prop      | Type                                     | Description                                                          | Default   |
+| --------- | ---------------------------------------- | -------------------------------------------------------------------- | --------- |
+| id?       | string                                   | Sets the id of the date picker. Should match the htmlFor of `Label`. | null      |
+| name?     | string                                   | Sets the name of the date picker.                                    | null      |
+| value?    | string                                   | Sets the value of the date picker.                                   | null      |
+| disabled? | boolean                                  | Disables the date picker.                                            | false     |
+| readOnly? | boolean                                  | Sets the date picker to readonly.                                    | false     |
+| required? | boolean                                  | Sets the date picker as required.                                    | false     |
+| hasError? | boolean                                  | Sets the date picker to an error state.                              | false     |
+| min?      | string                                   | Sets the earliest valid date value.                                  | null      |
+| max?      | string                                   | Sets the last valid date value.                                      | null      |
+| onChange? | `(event: React.MouseEvent<HTMLElement>)` |                                                                      | null      |
+| onFocus?  | `(event: React.MouseEvent<HTMLElement>)` |                                                                      | null      |
+| onBlur?   | `(event: React.MouseEvent<HTMLElement>)` |                                                                      | null      |
+| variant?  | 'default', 'inverse'                     |                                                                      | 'default' |
+
+#### formatReturnDate() Props
+
+| Prop       | Type   | Description                                                                                                           | Default |
+| ---------- | ------ | --------------------------------------------------------------------------------------------------------------------- | ------- |
+| dateValue  | string | Date value to be formatted. Must be in <inlineCode>yyyy-mm-dd</inlineCode> format.                                    | null    |
+| dateFormat | string | Speficy the format in which to return the dateValue. Formats come from the <inlineCode>date-fns</inlineCode> library. | null    |
+
+#### Label Props
```

Comment:
Since Label and HelpText are separate components, would it make more sense to have these prop tables live on those doc pages? I can see that both those doc pages are empty, so this might be a topic for another day haha.

Reply:
Yeah, I was talking to @SiTaggart about this recently. I think it comes from when `Form` was it's own package that exported Input, Label, HelpText, etc. Might make sense to leave it there for now (to be consistent with the Input page) until we rework the component pages. What do you think?